syntax = "proto3";
package kv_operate;

// The KV service definition.
service KV_server {
  rpc Add_KV (Add_Req) returns (Reply) {}
  rpc Delete_KV (Delete_Req) returns (Reply) {}
  rpc Query_KV (Query_Req) returns (Reply) {}
}

message Add_Req {
  string kv_json = 1;
}

message Delete_Req {
  string key = 1;
}

message Query_Req {
  string key = 1;
}

message Reply {
  string res = 1;
}